.editorconfig

```diff
@@ -12,7 +12,7 @@ indent_style = space
 trim_trailing_whitespace = true
 
 # MSVC specific
-cpp_generate_documentation_comments = doxygen_triple_slash
+cpp_generate_documentation_comments = none
```

Comment:
Why this change?

Reply:
Because the default then is that when you type /// VS will auto complete it to /// @brief.
That was also the intention. However, we use /// \brief. So no I always need to modify it again, which is annoying.
So either we agree that using @ is also fine, then it is useful, or we just continue doing it manually. And I didn't have the energy for yet another discussion about these kind of things, I had enough of them the last weeks :D

Reply:
Well Ok I have no strong feeling one way or another. We could just use `@`, the end result will be the same.